src/SourceDialog/SourceDialog.vala

```diff
@@ -186,31 +204,39 @@ public class Maya.View.SourceDialog : Gtk.Grid {
             set_as_default = !set_as_default;
         });
 
-        color_button_red.toggled.connect (() => {
-            hex_color = "#da3d41";
+        color_button_strawberry.toggled.connect (() => {
+            hex_color = "#ed5353";
         });
 
         color_button_orange.toggled.connect (() => {
-            hex_color = "#f37329";
+            hex_color = "#ffa154";
+        });
+
+        color_button_banana.toggled.connect (() => {
+            hex_color = "#f9c440";
         });
 
-        color_button_yellow.toggled.connect (() => {
-            hex_color = "#e6a92a";
+        color_button_lime.toggled.connect (() => {
+            hex_color = "#68b723";
         });
 
-        color_button_green.toggled.connect (() => {
-            hex_color = "#81c837";
+        color_button_mint.toggled.connect (() => {
+            hex_color = "#28bca3";
```

Comment:
For WCAG compliance we would need a background of at least @MINT_700 with an assumed foreground color of #fff.